BSP/STM32F413/Src/BSP_Contactor.c

```diff
@@ -4,39 +4,117 @@
 #include "stm32f4xx.h"
 
 /**
- * @brief   Initializes the GPIO pins that interfaces with the Contactor.
+ * Used pins:
+ * A:
+ * 		PA2, PA3 - ADC and UART
+ * 		PA8 - I2C SCL
+ * 		PA4 thru PA7 - Lights
+ * 		PA15 - SPI CS
+ * B:
+ * 		PB3 thru PB5 - SPI
+ * 		PB8, PB9 - CAN
+ * 		PB10 - UART
+ * 		PB12 - Lights
+ * 		PB14, PB15 - Fans
+ * C:
+ * 		PC5 - also UART
+ * 		PC6, PC7 - Fans
+ * 		PC9 - I2C SDA
+ * 		PC0 thru PC4 - Lights
+ * 		PC10 thru PC12 - more SPI
+ * D:	
+ * 		PD2 - SPI CS
+ * 		
+ * 
+ * 		Available Pins for contactors:
+ * 		PB0-1 used for C1
+ * 		PC0-1 can use for C3
+ * 		PA4-5 can use for C2
+ */
```

Comment:
I don't think these comments should go here. Each library should have comments for its own hardware support.